backend/src/controllers/tasks/index.ts

```diff
@@ -74,13 +75,43 @@ export const updateTask = async (req: Request, res: Response) => {
   }
 };
 
-export const getProject = async (req: Request, res: Response) => {
+export const assignTaskToUser = async (req: Request, res: Response) => {
   const { project_id } = req.params;
+  const { assignee, title, description, status, completed, priority, due_date } = req.body; // Assuming the task details are in the request body
+
   try {
```

Comment:
You are both creating the task and assigning a user. Wouldn't it be better to be split up?